docs/plugin/plugin.md

```diff
@@ -27,43 +27,260 @@ Options:
 ### Operator Deletion
 Command: `kubectl rw uninstall`
 
-Uninstall the RisingWave Operator from the cluster
+Uninstall the RisingWave Operator from the cluster.
 
 ## Basic Commands
 
-#### RisingWave Instance Creation
+### RisingWave Instance Creation
 
 Command: `kubectl rw create INSTANCE_NAME [options]`
 
-Create a risingwave named example-rw by configuration file.
+Create a RisingWave instance.
 
-example: `kubectl rw create example-rw -n test -c example.toml`
+Examples:
+
+- Create a risingwave named example-rw in the test namespace by default configuration.
+
+`kubectl rw create example-rw -n test`
+
+- Create a risingwave named example-rw by configuration file.
+
+`kubectl rw create -c rw.config`
 
 Options:
 
-- `--namespace=risingwave`
-- `--config=example.toml`
+- `-a, --arch=''` The default arch(will be override if config file also set the arch).
```

Comment:
Oh, does the `arch` option still work now?

Reply:
the `arch` is no longer necessary and I will create an issue to remove it